superset/assets/javascripts/explorev2/stores/controls.jsx

```diff
@@ -991,7 +991,7 @@ export const controls = {
     description: 'Overlay a timeseries from a ' +
                  'relative time period. Expects relative time delta ' +
                  'in natural language (example:  24 hours, 7 days, ' +
-                 '56 weeks, 365 days',
+                 '56 weeks, 365 days)',
```

Comment:
sorry for my misunderstanding, I just saw there is a left one in line 993, thus thought so :P